python/ray/worker.py

```diff
@@ -299,12 +311,12 @@ def store_and_register(self, object_id, value, depth=100):
                     value,
                     object_id=pyarrow.plasma.ObjectID(object_id.id()),
                     memcopy_threads=self.memcopy_threads,
-                    serialization_context=self.serialization_context)
+                    serialization_context=self.get_serialization_context(self.task_driver_id.__hash__()))
```

Comment:
Let's remove all of the calls to `__hash__`. What are they for?

Reply:
explained as above

Reply:
What I mean is, why do you need to call `__hash__()`, why not just directly pass in the ID?